app/models/katello/docker_tag.rb

```diff
@@ -30,6 +38,16 @@ def update_from_json(json)
       self.save!
     end
 
+    def self.import_all(uuids = nil, options = {})
```

Comment:
@beav 
It is not very straightforward unfortunately :(.
Basically with schema 2, one is going to have the same tag name for 2 different manifests in the same repo (one for schema1 and other for schema 2). Pulp just imports every thing from the upstream repo.
However the user is likely to think in terms of Tags per repo. 
We created this new model DockerMetaTag to address that issue. Basically DockerMetaTag Model has schema1 and schema2 attributes  that reference the schema1/schema2 docker tags.
DockerMetaTag gets created after DockerTags are indexed 
The other thing to note is that "index/show" methods of docker tags controller now point to DockerMetaTag instead (so as to group the tags with the same name in a repo).